web/blocks.html

```diff
@@ -26,6 +26,7 @@
     <link rel="stylesheet" href="css/codeworld.css">
     <link rel="stylesheet" href="css/show-hint.css">
     <link rel="stylesheet" href="css/codeworld-cm.css">
+    <link rel="stylesheet" href="css/directoryTree.css" />
```

Comment:
[nit] Change this everywhere else, if you do it here.

Reply:
Change what? Insert slash everywhere?

Reply:
Slash is removed.